### PR TITLE
Update serverless pricing info 

### PR DESCRIPTION
**Pull request by marciw**
Fix elastic/platform-docs-team#492 by updating pricing info as approved by @linyaru in the associated [Gdoc]( https://docs.google.com/document/d/1wzdbQAIhqyGZ66SOjXYmmITbmJKFJG3yqBjIsWH48qI/edit?tab=t.0#heading=h.h40htj7o6ooy). Also make minor edits for style and clarity.

**Previews**
- [ES billing dimensions](https://docs-content_bk_201.docs-preview.app.elstc.co/guide/en/serverless/current/elasticsearch-billing.html)
- [Manage your project - Search Power settings](https://docs-content_bk_201.docs-preview.app.elstc.co/guide/en/serverless/current/elasticsearch-manage-project.html#elasticsearch-manage-project-search-power-settings)
- [Monitor your account usage](https://docs-content_bk_201.docs-preview.app.elstc.co/guide/en/serverless/current/general-monitor-usage.html)

---
Notes:
- Needs a later pass to comprehensively apply the new variables in #196 
- Additional edits would also be nice; did only the essentials for now

### CODE REVIEW

File: serverless/pages/pricing.asciidoc [hunk 1]
```diff
@@ -6,51 +6,48 @@
...
-== Information about the Search AI Lake dimension (GB)
+== Search AI Lake dimension (GB)
 
-For Elasticsearch, the Search AI Lake is where data is stored and retained. This is
-charged in GBs for the size of data at rest. Depending on the enrichment,
-vectorization and other activities during ingest, this size may be different
+Elasticsearch stores and retains data in the <<elasticsearch-manage-project-search-ai-lake-settings,Search AI Lake>>, 
+which is charged in GB for the size of data at rest. Depending on enrichment,
+vectorization, and other activities during ingest, this size may be different
 from the original size of the source data.
```
Comment (leemthompo) [inline serverless/pages/pricing.asciidoc:38]:
```suggestion
{es-serverless} projects store data in the <<elasticsearch-manage-project-search-ai-lake-settings,Search AI Lake>>. You are charged per GB of data at rest. Note that if you perform operations at ingest such as vectorization or enrichment, your storage size will differ from your original source data.
```

Suggestion to be more direct

**Review by leemthompo — APPROVED** [review]
LGTM, with a few suggestions :)

File: serverless/pages/pricing.asciidoc [hunk 2]
```diff
@@ -6,51 +6,48 @@
...
-* **Machine Learning** — The VCUs used to perform inference, NLP tasks, and other ML activities.
+* **Indexing:** The VCUs used to index incoming documents.
+* **Search:** The VCUs used to return search results, with the latency and
+queries per second (QPS) you require.
+* **Machine learning:** The VCUs used to perform inference, NLP tasks, and other ML activities.
 
 [discrete]
 [[elasticsearch-billing-information-about-the-search-ai-lake-dimension-gb]]
-== Information about the Search AI Lake dimension (GB)
+== Search AI Lake dimension (GB)
```
[the remark this answers is not in the document]
Comment (leemthompo) [reply serverless/pages/pricing.asciidoc:33]: something like "How data is stored and billed"

File: serverless/pages/pricing.asciidoc [hunk 3]
```diff
@@ -6,51 +6,48 @@
...
+as well as the number of searches and the latency you require for searches. In addition, if
+you use {ml} for inference or NLP tasks, those VCUs are also
 metered and billed.
 
-include::../partials/minimum-vcus-detail.asciidoc[]
-
 [discrete]
 [[elasticsearch-billing-information-about-the-vcu-types-search-ingest-and-ml]]
-== Information about the VCU types (Search, Ingest, and ML)
+== VCU types: Search, Ingest, and ML
```
Comment (florent-leborgne) [inline serverless/pages/pricing.asciidoc:22]:
```suggestion
== VCU types: Search, Indexing, and ML
```
To match with the items below?

File: serverless/pages/pricing.asciidoc (hunk 1 above)
Comment (florent-leborgne) [reply serverless/pages/pricing.asciidoc:38]: We might want an extra explicit statement here that says which of the 2 gets billed, or reword slightly the last sentence in the suggestion

File: serverless/pages/pricing.asciidoc [hunk 4]
```diff
@@ -6,51 +6,48 @@
...
-
-For detailed Elasticsearch serverless project rates, check the
-https://www.elastic.co/pricing/serverless-search[Elasticsearch Serverless pricing page].
+You can control costs by using a lower Search Power setting or reducing the amount 
+of retained data. 
+
+* **Search Power setting:** <<elasticsearch-manage-project-search-power-settings,Search Power>> controls the speed of searches against your data. With Search Power, you can 
+improve search performance by adding more resources for querying, or you can reduce provisioned 
+resources to cut costs.
+* **Time series data:** By limiting the number of days of <<elasticsearch-ingest-time-series-data,time series data>> that are available for caching,
```
Comment (florent-leborgne) [inline serverless/pages/pricing.asciidoc:50]:
```suggestion
* **Time series data retention:** By limiting the number of days of <<elasticsearch-ingest-time-series-data,time series data>> that are available for caching, 
```

File: serverless/pages/pricing.asciidoc (hunk 1 above)
Comment (leemthompo) [reply serverless/pages/pricing.asciidoc:38]: I just tweaked to say "stored data" in both sentences, is that clearer?

File: serverless/pages/pricing.asciidoc [hunk 5]
```diff
@@ -6,51 +6,48 @@
...
-infrastructure used to support your use case, with the performance
-characteristics you need. We measure by Virtual Compute Units (VCUs), which is a
-slice of RAM, CPU and local disk for caching. The number of VCUs required will
-depend on the amount and the rate of data sent to Elasticsearch and retained,
-and the number of searches and latency you require for searches. In addition, if
-you required {ml} for inference or NLP tasks, those VCUs are also
+Elasticsearch is priced based on consumption of the underlying
+infrastructure that supports your use case, with the performance
+characteristics you need. Measurements are in Virtual Compute Units (VCUs).
+Each VCU represents a slice of RAM, CPU, and local disk for caching.
```
Comment (leemthompo) [inline serverless/pages/pricing.asciidoc:12]:
```suggestion
A VCU measures consumption of RAM, CPU, and local disk for caching.
```

I understand a lot of these edits are pre-existing text 😄  🍕

File: serverless/pages/pricing.asciidoc [hunk 6]
```diff
@@ -6,51 +6,45 @@
...
+Elasticsearch is priced based on consumption of the underlying
+infrastructure that supports your use case, with the performance
+characteristics you need. Measurements are in Virtual Compute Units (VCUs).
+Each VCU represents a fraction of RAM, CPU, and local disk for caching.
+
+The number of VCUs required 
+depends on the amount and rate of data sent to Elasticsearch and retained,
+as well as the number of searches and the latency you require for searches. In addition, if
+you use {ml} for inference or NLP tasks, those VCUs are also
 metered and billed.
```
Comment (leemthompo) [inline serverless/pages/pricing.asciidoc:18]:
```suggestion
The number of VCUs you need is determined by:

* Volume and ingestion rate of your data
* Data retention period 
* Search query volume
* Search Power setting
* Machine learning usage (if applicable):
```

File: serverless/pages/pricing.asciidoc [hunk 7]
```diff
@@ -6,51 +6,45 @@
...
-* **Machine Learning** — The VCUs used to perform inference, NLP tasks, and other ML activities.
+* **Indexing:** The VCUs used to index incoming documents.
+* **Search:** The VCUs used to return search results, with the latency and
+queries per second (QPS) you require.
+* **Machine learning:** The VCUs used to perform inference, NLP tasks, and other ML activities.
 
 [discrete]
 [[elasticsearch-billing-information-about-the-search-ai-lake-dimension-gb]]
-== Information about the Search AI Lake dimension (GB)
+== Search AI Lake dimension (GB)
```
Comment (leemthompo) [inline serverless/pages/pricing.asciidoc:33]:
```suggestion
== Data storage and billing
```

File: serverless/pages/pricing.asciidoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (georgewallace) [reply serverless/pages/pricing.asciidoc]: took your other suggestion.

**Comment by georgewallace** [comment]
run docs-build

**Comment by georgewallace** [comment]
@leemthompo since @marciw was out, I worked on accepting your feedback and fixing some merge conflicts. If this looks good can you sign off when you get in. I think I got everything.

cc: @marciw

File: serverless/pages/manage-billing-monitor-usage.asciidoc [hunk 8]
```diff
@@ -6,23 +6,18 @@
...
+On the **Usage** tab of the **Billing** page, you can:
 
-* Monitor the usage for the current month, including total hourly rate and month-to-date usage
+* Monitor usage for the current month, including month-to-date usage
 * Check the usage breakdown for a selected time range
+* View usage totals by product
 
 [IMPORTANT]
 ====
 The usage breakdown information is an estimate. To get the exact amount you owe for a given month, check your invoices in the <<general-billing-history,billing history>>.
```
Comment (leemthompo) [inline serverless/pages/manage-billing-monitor-usage.asciidoc:22]:
```suggestion
The usage breakdown information is an estimate. To get the exact amount you owe for a given month, check your invoices in the <<general-billing-history,billing history>>.
====
```

**Review by florent-leborgne — APPROVED** [review]

**Pull request merged**